docs/FAQ.rst

```diff
@@ -182,7 +160,7 @@ An alternative, if multithreading is really necessary inside the forked sessions
 Intel toolchain. Intel compilers are unaffected by this bug.
 
 For C/C++ users, any OpenMP feature cannot be used before the fork happens. If an OpenMP feature is used before the
-fork happens (ex: using OpenMP for forking), OpenMP will hang inside the forked sessions. Use new processes instead
+fork happens (example: using OpenMP for forking), OpenMP will hang inside the forked sessions. Use new processes instead
```

Comment:
this is a good change, I think shortened words like this can be a bit unfriendly to users for whom English is not their first language